be/src/olap/cumulative_compaction_time_series_policy.cpp

```diff
@@ -70,6 +70,12 @@ uint32_t TimeSeriesCumulativeCompactionPolicy::calc_cumulative_compaction_score(
         return score;
     }
 
+    // If their are many empty rowset, maybe should be compact
+    auto consecutive_empty_rowsets = tablet->collect_consecutive_empty_rowsets();
```

Comment:
maybe we should treat it as condition 4 and put it in the last

Reply:
done